app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -278,13 +278,17 @@ class Core(private val context: Context) {
     }
 
     private fun geCustomCookiePolicy(): CookiePolicy {
-            return when (context.settings().blockCookiesSelectionInCustomTrackingProtection) {
+        return if (context.settings().blockCookiesInCustomTrackingProtection) {
```

Comment:
I think we need to improve the logic here as we only want to set `CookiePolicy.ACCEPT_ALL` when in the  ETP UI the **Custom Radio Button** is marked and the **Cookies checkbox** is unchecked this is going to return true every time the checkbox is checked, and by default `blockCookiesInCustomTrackingProtection` [is set to true](https://github.com/mozilla-mobile/fenix/blob/388c7017d8378500eb986e5473465572055d804a/app/src/main/java/org/mozilla/fenix/utils/Settings.kt#L245)